services/scan/src/precinct_scanner_state_machine.ts

```diff
@@ -795,15 +805,35 @@ export function createPrecinctScannerStateMachine(
         }
       })();
       const { error, interpretation } = state.context;
+      // Remove interpretation details
```

Comment:
the frontend doesn't actually need to know the raw interpretation data, just the `type` and associated `reason`/`reasons`